src/pageObject.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
 class PageObject {
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
 
+  clickOnLogo() {
+    return `Click on the ${this.header.logo}`;
+  }
 };
 
 class ArticlePage extends PageObject {
+  constructor(url, slug) {
+    super(url);
+    this.url += slug;
```

Comment:
The assignment to `this.url` on line 23 overrides the `url` property set by the `super(url)` call. Consider using a different property name for the combined URL or ensure this behavior is intentional and won't cause issues in other parts of the code.